<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prefsDialog</class>
 <widget class="QDialog" name="prefsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SFBM Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="settingsTab">
        <attribute name="title">
         <string>Settings</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QVBoxLayout" name="trayIconLayout">
           <item>
            <widget class="QLabel" name="trayiconLabel">
             <property name="text">
              <string>Tray Icon:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="trayiconButton">
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="consoleLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="consoleLabel">
               <property name="text">
                <string>Console:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="consoleComboBox"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="consoleLineEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="directoryTab">
        <attribute name="title">
         <string>Directories</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="listLayout">
             <item>
              <widget class="QListView" name="listView"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="listButtonsLayout">
               <item>
                <layout class="QHBoxLayout" name="addRemoveLayout">
                 <item>
                  <widget class="QPushButton" name="addButton">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset theme="list-add">
                     <normaloff/>
                    </iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="removeButton">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset theme="list-remove">
                     <normaloff/>
                    </iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QHBoxLayout" name="upDownLayout">
                 <item>
                  <widget class="QPushButton" name="upButton">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset theme="arrow-up">
                     <normaloff/>
                    </iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="downButton">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset theme="arrow-down">
                     <normaloff/>
                    </iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="optionsLayout">
             <item alignment="Qt::AlignLeft">
              <widget class="QLabel" name="iconLabel">
               <property name="text">
                <string>Icon:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QPushButton" name="iconButton">
               <property name="minimumSize">
                <size>
                 <width>36</width>
                 <height>36</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>36</width>
                 <height>36</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="optionsLabel">
               <property name="text">
                <string>Options:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="dirsFirstBox">
               <property name="text">
                <string>Directories First</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="showHiddenBox">
               <property name="text">
                <string>Show Hidden</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="includePreviousBox">
               <property name="text">
                <string>Include '..'</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="closeLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
         <property name="icon">
          <iconset theme="dialog-ok">
           <normaloff/>
          </iconset>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
